frontend/packages/pipelines-plugin/integration-tests/support/pages/pipelines/task-page.ts

```diff
@@ -0,0 +1,52 @@
+import * as yamlEditor from '@console/cypress-integration-tests/views/yaml-editor';
+import { pipelineBuilderPO } from '../../page-objects/pipelines-po';
+
+export const tasksPage = {
+  clickOnCreateTask: () => {
+    cy.get('[data-test-id="dropdown-button"]').click();
+    cy.get('[data-test-dropdown-menu="tasks"]').click();
+    yamlEditor.isLoaded();
+  },
+  clearYAMLEditor: () => {
+    cy.get(pipelineBuilderPO.yamlView.yamlEditor)
+      .click()
+      .focused()
+      .type('{ctrl}a')
+      .clear();
+  },
+  setEditorContent: (yamlLocation: string) => {
+    cy.readFile(yamlLocation).then((str) => {
+      yamlEditor.setEditorContent(str);
+    });
+  },
+  openTasksInPipelinesSidebar: () => {
+    cy.get('[data-test=nav]')
+      .contains('Pipelines')
+      .click();
+    cy.get('[data-test=nav]')
+      .contains('Tasks')
+      .click();
+  },
+  submitTaskYAML: () => {
+    cy.get('[data-test=save-changes]').click();
+  },
+  openPipelineSidebar: () => {
+    cy.get('[data-test=nav]')
+      .contains('Pipelines')
+      .click();
+  },
+  closePipelineSidebar: () => {
+    cy.get('[data-test=nav]')
+      .contains('Pipelines')
+      .click();
+  },
```

Comment:
Both are Identical functions that clicks on the  `Pipelines` Navigation Item, but are named differently. I looked at the code we have something like `navigateTo` function, shall we reuse that ? 

eg: navigateTo(devNavigationMenu.Pipelines);